jdaviz/app.py

```diff
@@ -148,6 +149,10 @@ def __init__(self, configuration=None, *args, **kwargs):
         # Parse the yaml configuration file used to compose the front-end UI
         self.load_configuration(configuration)
 
+        # If true, link data on load. If false, do not link data to speed up
+        # data loading
+        self.auto_link = kwargs.pop('auto_link', True)
```

Comment:
This doesn't actually work properly is ``auto_link`` is passed to the initializer as the ``auto_link`` option is first passed to the parent class in the ``super().__init__`` call and is then not recognized. For this to work it has to come before the ``super()`` call I think.